db-scripts/src/main/resources/migration.sql

```diff
@@ -457,3 +457,23 @@ ALTER TABLE gistic_to_gene ADD CONSTRAINT `gistic_to_gene_ibfk_2` FOREIGN KEY (`
 ALTER TABLE `mutation_event` MODIFY COLUMN `KEYWORD` VARCHAR(255);
 ALTER TABLE `mutation_count_by_keyword` MODIFY COLUMN `KEYWORD` VARCHAR(255);
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.6.1";
+
+##version: 2.7.0
+INSERT INTO `clinical_attribute_meta` SELECT 'MUTATION_COUNT', 'Mutation Count', 'Mutation Count', 'NUMBER', 0, '1', 
```

Comment:
Please find out the priority value that we should use.

Reply:
@jjgao @ersinciftci according to https://github.com/cBioPortal/cbioportal/blob/master/docs/Study-View.md, let's set Mutation Count as 30, FGA as 20